lib/seek/content_split.rb

```diff
@@ -0,0 +1,17 @@
+module Seek
+  module ContentSplit
+    def split_content(str, length = 10, overlap = 5)
```

Comment:
any reason for choosing 10 and 5 as default parameters?

Reply:
There is really no solid reason for this other than ease of use, I will remove the defaults